models/room.js

```diff
@@ -27,22 +27,19 @@ const roomSchema = new mongoose.Schema({
   },
 })
 
-const Room = mongoose.model('Room', roomSchema)
-
 const validateRoom = (room) => {
   const schema = Joi.object({
     hotelId: Joi.objectId().required(),
```

Comment:
![image](https://user-images.githubusercontent.com/38506213/109889321-6c09ac80-7c85-11eb-9f7c-c84cfbb39e86.png)


instead roomNumber like in schema